src/content/apply/apply.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Join Our Team",
   "subtitle": "Help us transform software development at Northeastern",
-  "applicationStatus": "Fall 2021 Applications are open",
-  "isBannerVisible": true
+  "applicationStatus": "Spring 2022 Applications are open",
```

Comment:
wait this is fake news, apps aren't even open yet LOL

Reply:
oh wait unless this doesn't show with the `isBannerVisible: false`

Reply:
ya lol it doesnt show I just updated for when it turns back to true

Reply:
thinking ahead bro